### PR TITLE
Fancy output

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

File: Linux/tree-armv7l/init [hunk 1]
```diff
@@ -1,39 +1,65 @@
 #!/bin/sh
 
+# Loading functions
+. /functions
+
+/bin/busybox sleep .5
```
Comment (aimxhaisse) [inline Linux/tree-armv7l/init]: why?

Comment (moul) [inline Linux/tree-armv7l/init]: to avoid some `printk` during printing of the banner

**Comment by aimxhaisse** [comment]
LGTM

**Pull request merged**